client/src/SharedComponents/CohortCard/CohortCard.js

```diff
@@ -1,14 +1,13 @@
 import React,{Component} from 'react';
 import "./CohortCard.css";
 
-class CohortCard extends Component {
-  render() {
+  const CohortCard = (props) => {
```

Comment:
@bayan-404 If you want, you can destructure arguments like this:
`const CohortCard = ({ name }) => {`